### PR TITLE
docs: add governance and socials information

### LINKED ISSUE #8: Add `GOVERNANCE.md`
> A GOVERNANCE file lets people know about how your project is governed. For example, it might discuss project roles and how decisions are made.
>
> \- [Supported file types - GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

The [community page](https://catppuccin.com/community) on the newly merged [website](https://catppuccin.com) contains information on the teams leading Catppuccin. It feels like the `GOVERNANCE` file is the ideal place for that information. 

To make this information machine readable, I'll probably opt for a similar workflow to catppuccin/catppuccin where I'll store the team information in a yaml file, validated by a simple schema and generate the markdown file. This will allow the website to retrieve leadership information similar to the ports/userstyles.

### PR DESCRIPTION
**Pull request by sgoudham**
This PR adds a `GOVERNANCE.md` which is generated from the script files inside
of the `governance/` folder.

A `socials/` folder has also been added to list all of the places where
Catppuccin is available on various social media platforms.

This will allow programmatic retrieval of governance information and socials
information, ideally for the Catppuccin website.

A reminder what the `GOVERNANCE.md` file should be for:

> A GOVERNANCE file lets people know about how your project is governed. For example, it might discuss project roles and how decisions are made.
>
> \- [Supported file types - GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

Closes #8

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<div align="center">
+  <h2>🏳️‍🌈 Governance</h2>
```
Comment (sgoudham) [inline GOVERNANCE.md]: Open to suggestions on the emoji

File: governance/governance.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    past-members: []
+  - name: "Moderator"
+    description: "Moderators of the Discord server."
+    responsibilities: "Responsible for moderating the community, including enforcing the rules and our code of conduct."
+    current-members:
+      - name: Nyx
+        url: https://github.com/nyxkrage
+      - name: Etzelia
+        url: https://github.com/jolheiser
+    past-members: []
```
Comment (sgoudham) [inline governance/governance.yml:51]: Reminder for myself: need to add a yaml schema comment before merging.

File: socials/socials.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  tiktok:
+    name: "TikTok"
+    url: "https://tiktok.com/@catppuccin"
+  spotify:
+    name: "Spotify"
+    url: "https://open.spotify.com/user/31elbnluryieb6lh33o4oluv4uqy?si=90eab9fb20d1485a"
+  linkedin:
+    name: "LinkedIn"
+    url: "https://www.linkedin.com/company/catppuccin/"
+# yaml-language-server: $schema=./socials.schema.json
```
Comment (sgoudham) [inline socials/socials.yml]: Reminder for myself: need to update to point to the main branch before merging

**Review by backwardspy — APPROVED** [review]
govern it 👍

**Review by nullishamy — APPROVED** [review]

**Comment by nullishamy** [comment]
make me core and then merge it

**Pull request merged**